webapp/src/ts/services/select2-search.service.ts

```diff
@@ -177,16 +183,25 @@ export class Select2SearchService {
     // Hydrate and re-set real doc on change
     // !tags -> only support single values, until there is a use-case
     if (!options.tags) {
-      selectEl.on('select2:select', (e) => {
-        const docId = e.params && e.params.data && e.params.data.id;
-
-        if (docId) {
-          this.getDoc(docId)
-            .then(doc => {
-              this.setDoc(selectEl, doc);
-              selectEl.trigger('change');
-            })
-            .catch(err => console.error('Select2 failed to get document', err));
+      let search: string | undefined;
+      selectEl.on('select2:selecting', () => search = selectEl.data('select2').dropdown.$search.val());
```

Comment:
@jkuester since the search input gets cleared before the `select2:select` event is called, this is the next best place to get its content before it gets cleared

Reply:
> search input gets cleared before the select2:select

That is super annoying, but yeah, in that case, this seems like a fine approach!  I assume `select2:selecting` is just getting repeatedly triggered as the user types in the search box?

Reply:
No that's the good part, it gets triggered right before a result is selected aka right before the `select2:select` event we have a listener for one line below.